pyreleaseplugin/release.py

```diff
@@ -168,11 +169,18 @@ def build():
         raise RuntimeError("Error building wheel")
 
 
-def publish_to_pypi():
+def publish_to_pypi(repository):
     """
     Publish the distribution to our local PyPi.
+    :param repository: the repository to be used by the ReleaseCommand for the artifact upload. If ommitted, repository will be defaulted to PyPi
```

Comment:
We use the style specified [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

So this should be
```python
"""
Args:
    repository: the repository to be uploaded to (eg. Artifactory or Pypi.org).
"""
```